akka-sample-cluster-java/README.md

```diff
@@ -21,27 +21,27 @@ The small program together with its configuration starts an ActorSystem with the
 
 You can read more about the cluster concepts in the [documentation](http://doc.akka.io/docs/akka/2.6/java/cluster-usage.html).
 
-To run this sample, type `sbt "runMain sample.cluster.simple.SimpleClusterApp"` if it is not already started. If you are using maven, use `mvn compile exec:java -Dexec.mainClass="sample.cluster.simple.SimpleClusterApp"` instead. To pass parameters to maven add `-Dexec.args="..."`.
+To run this sample, type `sbt "runMain sample.cluster.simple.App"` if it is not already started. If you are using maven, use `mvn compile exec:java -Dexec.mainClass="sample.cluster.simple.App"` instead. To pass parameters to maven add `-Dexec.args="..."`.
 
 `SimpleClusterApp` starts three actor systems (cluster members) in the same JVM process. It can be more interesting to run them in separate processes. Stop the application and then open three terminal windows.
 
 In the first terminal window, start the first seed node with the following command:
 
-    sbt "runMain sample.cluster.simple.SimpleClusterApp 2551"
+    sbt "runMain sample.cluster.simple.App 2551"
```

Comment:
mvn pom.xml and run instructions, you can use this commit as template: https://github.com/akka/akka-samples/pull/141/commits/b54f1b608d38d4f1c49d6df2a023a8f4d256f5e5

Reply:
There already is a pom.xml in this sample, I'll update the readme references.